workspaces/ballerina/ballerina-extension/src/features/project/cmds/cmd-runner.ts

```diff
@@ -70,7 +70,10 @@ export enum BALLERINA_COMMANDS {
 }
 
 export enum PROJECT_TYPE {
-    SINGLE_FILE = "SINGLE_FILE_PROJECT", BUILD_PROJECT = "BUILD_PROJECT", BALR_PROJECT = "BALR_PROJECT"
+    SINGLE_FILE = "SINGLE_FILE_PROJECT",
+    BUILD_PROJECT = "BUILD_PROJECT",
+    BALR_PROJECT = "BALR_PROJECT",
+    WORKSPACE = "WORKSPACE_PROJECT"
```

Comment:
Are we using this?

Reply:
No. Only `SINGLE_FILE` is being used among these items. Add it there since the LS APIs are responing with these project types.